app/server/src/types.ts

```diff
@@ -22,5 +24,6 @@ export interface AppLocals {
     configReader: ConfigReader,
     defaultCodeReader: DefaultCodeReader,
     wodinConfig: WodinConfig,
-    wodinVersion: String
+    wodinVersion: String,
+    redis: Redis
```

Comment:
I am curious, why add redis to WodinConfig ?

It could be used directly in SessionStore, less parameter to worry about.

Reply:
This isn't the config, but the app properties set up by the server on initialisation, and then available to the controller to use. The session store could manage its own connection, but this way it doesn't have to.